src/bedrock.cr

```diff
@@ -0,0 +1,84 @@
+require "json"
+require "awscr-signer"
+require "base64"
+
+require "./eventstream"
+require "./client"
+
+module AWS
+  module BedrockRuntime
+    class Client < AWS::Client
+      SERVICE_NAME = "bedrock-runtime"
+      def initialize(
+          @access_key_id = AWS.access_key_id,
+          @secret_access_key = AWS.secret_access_key,
+          @region = AWS.region,
+          @endpoint = URI.parse("https://bedrock-runtime.#{region}.amazonaws.com"),
+        )
+        @signer = Awscr::Signer::Signers::V4.new("bedrock", region, access_key_id, secret_access_key)
```

Comment:
This was annoying to figure out. The signer name must be "bedrock" even though the SERVICE_NAME and endpoint are "bedrock-runtime".
I don't know how many AWS services have this quirk or if the abstraction to decouple these constants would be worse than this ugly override.

Reply:
Except bedrock and SES, these are minor services / minor supplemental features on major services that otherwise are well-behaved?
Eventually the decoupling is necessary, but maybe Not Yet. Perhaps worth sticking a note somewhere for the next person to add one of these services?